evap/settings.py

```diff
@@ -97,6 +97,10 @@
 if DEBUG:
     EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
+# Config for legal notice
+LEGAL_NOTICE_LINK = '' # e.g. "Impressum"
+LEGAL_NOTICE_FILE = '' # e.g. "impressum_text.html" which then has to be in evap\templates
```

Comment:
why is the setting called "legal notice" while the template and view name is "impressum"?